packages/php-table-backend-utils/src/Table/Snowflake/SnowflakeTableReflection.php

```diff
@@ -64,11 +72,18 @@ private function cacheTableProps(bool $force = false): void
         }
         $this->sizeBytes = (int) $row[0]['BYTES'];
         $this->rowCount = (int) $row[0]['ROW_COUNT'];
+        $this->isExternal = false;
+
         switch (strtoupper($row[0]['TABLE_TYPE'])) {
             case 'BASE TABLE':
                 $this->isTemporary = false;
                 $this->isView = false;
                 return;
+            case 'EXTERNAL TABLE':
+                $this->isTemporary = false;
+                $this->isView = false;
+                $this->isExternal = true;
```

Comment:
proč si interně nedržíš ten kind a je tam pořád isView a isExternal? Přijde mě to dost divné. (Chápu, že sis chtěl zkusit napsat match :D)